virttest/qemu_devices/qdevices.py

```diff
@@ -1104,6 +1123,10 @@ def verify_hotplug(self, out, monitor):
         else:
             return False
 
+    def _cmdline_json(self):
```

Comment:
hi Nick, do you think this function can cover the following situation?
 -device {"driver":"scsi-hd","bus":"scsi0.0","channel":0,"scsi-id":0,"lun":0,"device_id":"drive-scsi0-0-0-0","drive":"libvirt-1-format","id":"scsi0-0-0-0","bootindex":1,"write-cache":"on"}
As you can see, there are also some 'digits' inline. If we set the param with a digit instead of a string, I should be OK here. Please make a double check, thanks